src/hotspot/share/gc/shenandoah/shenandoah_globals.hpp

```diff
@@ -97,7 +97,7 @@
           "collector accepts. In percents of heap region size.")            \
           range(0,100)                                                      \
                                                                             \
-  product(uintx, ShenandoahOldGarbageThreshold, 10, EXPERIMENTAL,           \
+  product(uintx, ShenandoahOldGarbageThreshold, 15, EXPERIMENTAL,           \
           "How much garbage an old region has to contain before it would "  \
           "be taken for collection.")                                       \
```

Comment:
```
"An old region must contain at least this percentage of garbage before being eligible for evacuation."
```
What happens when the user specifies a number like 0 or 100? Does it have the requisite effect? For 0, I would expect the region to be eligible for evacuation even when there is no garbage in it and, for 100, I would expect it to be collected only when it is fully garbage and not otherwise. Is this a reasonable expectation? Are there tests that check for this?